docs/concepts/resources/filter_types.py

```diff
@@ -184,39 +184,52 @@
 y = pf.dsp.filter.low_shelve(impulse, 1/4*frequency, -10, 2, 'II')
 pf.plot.freq(y * 10**(-40/20), ax=axis, label='Low-shelve')
 
+# notch filter
+y = pf.dsp.filter.notch(impulse * .1**3, 100, 4)
+pf.plot.freq(y, ax=axis, label='Notch')
+
+y = pf.dsp.filter.notch(impulse * .1**4, 1000, 4)
+pf.plot.freq(y, ax=axis, label='Notch')
+
+y = pf.dsp.filter.notch(impulse * .1**5, 10000, 4)
+pf.plot.freq(y, ax=axis, label='Notch')
```

Comment:
I'd say one notch at 1kHz is enough here :)

Reply:
yes, true